xrootd/cmd/xrd-ls/main.go

```diff
@@ -156,5 +163,11 @@ func format(o io.Writer, root string, fi os.FileInfo, long bool) {
 		return
 	}
 
-	fmt.Fprintf(o, "%v\t %d\t %s\t %s\n", fi.Mode(), fi.Size(), fi.ModTime().Format("Jan 02 15:04"), fi.Name())
+	var name string
```

Comment:
I would initialize name with the result of fi.Name, and then test whether 'name' is empty (and assign 'root' to it)

Reply:
fine with me